app/controllers/charge-elements.controller.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+/**
+ * Controller for /charge-elements endpoints
+ * @module ChargeElementsController
+ */
+
+const AddLicencesToWorkflowService = require('../services/charge-elements/add-licences-to-workflow.service.js')
+
+async function workflowTimeLimitedLicences (request, h) {
```

Comment:
If we can, I favour naming the controller handler after the endpoint.

For example

```javascript
  path: '/health/airbrake',
  handler: HealthController.airbrake,

  path: '/health/database',
  handler: HealthController.database,

  path: '/health/info',
  handler: HealthController.info,
```

So, I'd suggest renaming this

```suggestion
async function timeLimited (request, h) {
```